<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>929</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Hydret1D-GUI  v1.2.5</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>hydret1D.ico</normaloff>hydret1D.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="PlotWidget" name="langView"/>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="play_lzeit">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../icons/resume.bmp</normaloff>../icons/resume.bmp</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_99">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>FPS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fps">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>60</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lzeitlcd">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
         <property name="intValue" stdset="0">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="lzeitslider">
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="sliderPosition">
          <number>0</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickInterval">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="show_simdatetime">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="displayFormat">
          <string>dd.MM.yyyy HH:mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="flat">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>WSP-Längsschnitt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mannings">
         <property name="text">
          <string>Mannings Wert</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">rType</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="strickler">
         <property name="text">
          <string>K-Strickler Wert</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">rType</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="rauEdit">
         <property name="text">
          <string>Rauheit bearbeiten:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="wspRau">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>RNI</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RNV1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RNV2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RNV3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RNV4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RNV5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RNV6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RNV7</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="undo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="redo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>redo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_93">
         <property name="text">
          <string>Gewässer ID:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="lang_ID">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QFrame" name="frame_15">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_19">
       <item row="2" column="2" colspan="3">
        <widget class="QListWidget" name="ls_listWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="3">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>WSP</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="hch"/>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="hch_browse">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="2" colspan="3">
        <widget class="QTableWidget" name="wrTable">
         <column>
          <property name="text">
           <string>WSP</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rauheit</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="4" column="2" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Plot Properties</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1390</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="savestart"/>
    <addaction name="closeall"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="checkver"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Modell</string>
    </property>
    <addaction name="run"/>
    <addaction name="reload"/>
   </widget>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="addWSPData"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatei"/>
   <addaction name="menuRun"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="savestart"/>
   <addaction name="separator"/>
   <addaction name="run"/>
   <addaction name="reload"/>
  </widget>
  <action name="settings">
   <property name="icon">
    <iconset>
     <normaloff>../icons/settings.ico</normaloff>../icons/settings.ico</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="closeall">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="run">
   <property name="icon">
    <iconset>
     <normaloff>../icons/runmodel.ico</normaloff>../icons/runmodel.ico</iconset>
   </property>
   <property name="text">
    <string>Run...</string>
   </property>
   <property name="iconText">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="checkver">
   <property name="text">
    <string>Updates...</string>
   </property>
  </action>
  <action name="savestart">
   <property name="icon">
    <iconset>
     <normaloff>../icons/savestart.bmp</normaloff>../icons/savestart.bmp</iconset>
   </property>
   <property name="text">
    <string>Save Start...</string>
   </property>
  </action>
  <action name="reload">
   <property name="icon">
    <iconset>
     <normaloff>../icons/reload.ico</normaloff>../icons/reload.ico</iconset>
   </property>
   <property name="text">
    <string>Reload...</string>
   </property>
  </action>
  <action name="addQData">
   <property name="text">
    <string>Q-Messung</string>
   </property>
  </action>
  <action name="addWSPData">
   <property name="text">
    <string>WSP-Messung</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rauEdit</sender>
   <signal>toggled(bool)</signal>
   <receiver>wspRau</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="lsType"/>
  <buttongroup name="rType"/>
 </buttongroups>
</ui>
